Docs/source/usage/parameters.rst

```diff
@@ -265,6 +265,16 @@ Overall simulation parameters
         In electromagnetic mode, this solver can be used to initialize the species' self fields
         (``<species_name>.initialize_self_fields=1``) provided that the field BCs are PML (``boundary.field_lo,hi = PML``).
 
+          * warpx.use_2d_slices_fft_solver (`bool`, default: 0): Select the type of Integrated Green Function solver.
+            If 0, solve Poisson equation in full 3D geometry.
+            If 1, solve Poisson equation in a quasi 3D geometry. In this case, the code performes many 2D Poisson solves
+            on all :math:`(x,y)` slices, each slice at a given :math:`z`.
```

Comment:
Maybe a more useful description would be: "This solver neglects the z derivatives in the Laplacian of the Poisson equation. This is often a good approximation for ultra-relativistic beams propagating along the z direction, with the `relativistic` solver. As a consequence, this solver does not need to do an FFT along the z direction, and instead uses only transverse FFTs (along x and y) at each z position (or z "slice")."